k2/csrc/fsa.h

```diff
@@ -11,6 +11,8 @@
 #include <utility>
 #include <vector>
 
+#include "k2/csrc/util.h"
+
 namespace k2 {
 
 using Label = int32_t;
```

Comment:
should we place `Weight` in fsa.h? There is no weight in an acceptor.

Reply:
I guess @danpovey worte this because there is another weighted fsa structure, 

https://github.com/danpovey/k2/blob/08145d03d6dac7fb442857227ab632158fbb4191/k2/csrc/weights.h#L61

Reply:
Let's remove the Weight typedef.  I prefer to remove Label and StateId typedefs for now too, as well.